webClient/src/app/editor/code-editor/monaco-settings/monaco-settings.component.html

```diff
@@ -41,16 +41,17 @@ <h2>Editor Settings</h2>
       </div>
     </div>
     <div class="button-row">
-      <button mat-raised-button class="menu-button" color="warn" (click)="resetToDefault()">Defaults</button>
-      <button mat-raised-button class="menu-button" color="primary" (click)="commitToConfigService()">Save</button>
+      <button mat-raised-button class="menu-button defaults-button" color="warn" (click)="resetToDefault()">Defaults</button>
+      <button mat-raised-button class="menu-button save-button" color="primary" (click)="commitToConfigService()">Save</button>
     </div>
   </div>
+  <div class="divider"></div>
   <div class="preview">
-    <h2 style="padding: 0.5em">JSON Preview</h2>
+    <h4 class="title-card">Advanced</h4>
```

Comment:
But its also a preview.

Reply:
The rest, I just changed the labels to something that I think aligns more with what I see in previous apps. "JSON" for example I don't know if everyone will know what that is, but maybe we should leave "Preview" in some way because it's a preview indeed like you said

Reply:
The buttons have been made to say "Preview"